hack/sync-ec-cli-tasks.sh

```diff
@@ -25,39 +25,50 @@ set -o pipefail
 set -o nounset
 
 EC_CLI_REPO_PATH="${1}"
+RELEASE_PREFIX="release-v"
 
-cp -r "${EC_CLI_REPO_PATH}/tasks" .
+# helper function to add tasks to a git branch
+add_tasks() {
+  local remote_branch=${1}
+  local branch=$(echo "$remote_branch" | cut -d'/' -f2)
```

Comment:
In bash string replacement can do this:

```suggestion
  local branch="${remote_branch#*/}"
```